### ISSUE TITLE
tqdm progress bar (may) causes chrome crash by using a lot of memory

### ISSUE THREAD

**Issue by xiaonanzzz** [googlecolab/colabtools#1708]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:

I'm using pytorch for training a model and tqdm for showing a progress bar. Sometimes, the memory usage of browser will keep increasing until it crashes. The actually output is not very large.

- Describe the expected behavior:
The memory usage of browser is normal. 

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome

- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):

Cannot reproduce it easily, it doesn't happen always.

**Comment by blois** [comment]
Do you know which progress bar this is?

For example notebook is the first, console is the second in:
https://colab.research.google.com/gist/blois/0edb04e592c16cbab8f0f9b1cff3595e/tqdm_notebook.ipynb#scrollTo=K9sPXp-rfV2U

**Comment by blois** [comment]
I'm not seeing any memory use growth when executing:

```python
from tqdm import tqdm_notebook
from time import sleep

for j in tqdm_notebook(range(10000), desc=f'loop'):
  sleep(0.01)
```

Sometimes browser memory use will climb if you have devtools open, or I would recommend trying with an incognito browser window as it could be caused by an extension.

**Comment by farhadinima75** [comment]
hi
@blois  his right. I've got this issue many times before and now it's occur again.

every time this issue happens, the only way is clearing outputs and rest runtimes. 
I think setting a step interval for TQDM may solve the problem at all.
Any help would be appreciated.

**Comment by farhadinima75** [comment]
Hi again.

I've found a temporary solution until colab developers find a final solution.

in tqdm object, we can set like this: `refresh=False`

>             tbar.set_description('TRAIN ({}) | Loss: {:.3f} | Acc {:.2f} mIoU {:.2f} | B {:.2f} D {:.2f} |'.format(
>                                               epoch, self.total_loss.average, 
>                                                pixAcc, mIoU,
>                                                self.batch_time.average, self.data_time.average), refresh=False)

and this:
>        tbar = tqdm(self.train_loader, ncols=130, miniters=int(50), maxinterval = 1000)

these options set interval prints to 50 steps between and save a lot of Memory and processing power.

**Comment by blois** [comment]
I *think* the issue was layout related where many messages could cause browser layout issues, freezing up the browser when they got long. The fix for this should now be available, please speak up if the issue persists.
